Engine/libs/lua/loslib.c

```diff
@@ -77,14 +77,15 @@
 
 
 static int os_execute (lua_State *L) {
-  const char *cmd = luaL_optstring(L, 1, NULL);
```

Comment:
Add a #ifdef flag instead of remove it completely

Reply:
got it